setup.py

```diff
@@ -5,12 +5,12 @@
 """
 
 from setuptools import setup, find_packages
-import pybert
 
 setup(
     name="PyBERT",
-    version=pybert.__version__,
```

Comment:
Hi @jdpatt , Can you tell me why you removed this version synching mechanism?

Reply:
I don't know where else you asked about the `setup.py` but the setup.py file is completely removed in the update tooling since `pyproject.toml` replaced it.  We can't run python code in a toml file, so it has to be got a different way.

When you are in a src structure, import pybert doesn't work and you get a module not found error.